src/Control/HTTP.php

```diff
@@ -88,7 +88,7 @@ public static function absoluteURLs($html)
      * for example:
      * <code>
      * function($url) {
-     *      return Director::absoluteURL($url, true);
+     *      return Director::absoluteURL((string) $url, true);
```

Comment:
Lets do `function(string $url)` here instead to promote good standards

Reply:
Updated